File: Atcoder/abc054/a.py
a,b = map(int,input().split( ))
if a ==1:
	a = 14
if b == 1:
	b =14

if a > b:
	ans = "Alice"
elif b > a:
	ans = "Bob"
elif a == b:
	ans = "Draw"

print(ans)

File: atcoder/abc054/a.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	int A,B;
	cin>>A>>B;
	if(A==1)A=14;
	if(B==1)B=14;
	string ans = "Bob";
	if(A>B)ans = "Alice";
	else if(A==B)ans ="Draw";
	cout<<ans<<endl;
}